docs/current-1.1/tools/dotnet-nuget-push.md

```diff
@@ -0,0 +1,116 @@
+---
+title: dotnet-nuget-push command | .NET Core SDK
+description: The dotnet-nuget-push command pushes a package to the server and publishes it. 
+keywords: dotnet-nuget-push, CLI, CLI command, .NET Core
+author: karann-msft
+ms.author: mairaw
+manager: wpickett
+ms.date: 11/10/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: f54d9adf-94f8-41cc-bb52-42f7ca3be6ff
+---
+
+#dotnet-nuget-push
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+## Name 
+dotnet-nuget-push -- Pushes a package to the server and publishes it. 
```

Comment:
put dotnet-nuget-push in a code block and remove the extra hyphen (similar to [dotnet-pack](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-pack))